pkg/test/ipfix.go

```diff
@@ -0,0 +1,114 @@
+package test
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/vmware/go-ipfix/pkg/entities"
+)
+
+const (
+	templateID          = 256
+	observationDomainID = 1
+)
+
+// Values taken from https://www.iana.org/assignments/ipfix/ipfix.xhtml
+var (
+	sourceIPv4Address = entities.NewInfoElement("sourceIPv4Address", 8, entities.Ipv4Address, 0, 4)
```

Comment:
This "feels" like end-to-end test ... and very specific to IPFIX ... do we need this ... what component is being tested here ???

Reply:
This is used to test the Ingest Collector (`ingest_collector_test.go`), as it remained untested.

Reply:
@mariomac ok ... this looks good to me  ... the only comment small comment I have is that maybe it makes sense to put this file, and maybe also the Loki mock file under some "internal" folder underneath "test" and not directly in test.